activemodel/lib/active_model/serialization.rb

```diff
@@ -31,9 +31,9 @@ module ActiveModel
   # of the attributes hash's keys. In order to override this behavior, take a look
   # at the private method +read_attribute_for_serialization+.
   #
-  # The JSON serialization is provided by default when you include the
-  # <tt>ActiveModel::Serialization</tt> module, so there is no need to explicitly
-  # include it.
+  # <tt>ActiveModel::Serializers::JSON</tt> module automatically includes
```

Comment:
Don't wrap things which could link in the docs. RDoc should automatically replace this with a link to the actual documentation for that module.